test/apiv2/40-pods.at

```diff
@@ -52,11 +61,16 @@ t POST "libpod/pods/fakename/unpause" 404\
   .cause="no such pod" \
   .message="no pod with name or ID fakename found: no such pod"
 
+t DELETE libpod/pods/foo?force=false 500 \
+  .cause~".*cannot remove container .* as it is running.*"
 
 t POST libpod/pods/foo/stop 200 \
   .Errs=null \
   .Id=$pod_id
 
+# Remove container as it is no longer required
+t DELETE libpod/containers/$cid 200 .[0].Id=$cid
+
```

Comment:
Hey, this is out of scope, and please ONLY if you need to repush... but... line 102 below, `sleep 999`, could you change it to `top` instead?

Reason: way way down, line 120, `DELETE /the/pod`, takes ten seconds because `sleep` is not interruptible. I tried adding `\&t=0` but that seems to be ignored by `pod delete`. Thank you!

Reply:
Done